### PR TITLE
feat: Publish runtime metadata in release

### PR DESCRIPTION
**Pull request by ukint-vs**
Resolves # .

-
-
-

@reviewer-or-team

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -66,6 +66,10 @@ jobs:
       - name: "Test: Production `vara-runtime`"
         run: ./wasm-proc --check-runtime-imports --check-runtime-is-dev false target/production/wbuild/vara-runtime/vara_runtime.compact.wasm
 
+      - name: "Build: Prod Runtime metadata"
```
Comment (breathx) [inline .github/workflows/release.yml:69]:
```suggestion
      - name: "Artifact: Production `vara-runtime` metadata"
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -75,6 +79,10 @@ jobs:
       - name: "Test: Development `vara-runtime`"
         run: ./wasm-proc --check-runtime-imports --check-runtime-is-dev true target/production/wbuild/vara-runtime/vara_runtime.compact.wasm
 
+      - name: "Build: Dev Runtime metadata"
```
Comment (breathx) [inline .github/workflows/release.yml:82]:
```suggestion
      - name: "Artifact: Development `vara-runtime` metadata"
```

**Review by breathx — APPROVED** [review]

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -66,6 +66,10 @@ jobs:
       - name: "Test: Production `vara-runtime`"
         run: ./wasm-proc --check-runtime-imports --check-runtime-is-dev false target/production/wbuild/vara-runtime/vara_runtime.compact.wasm
 
+      - name: "Build: Prod Runtime metadata"
+        run: |
+          RUNTIME_WASM=target/production/wbuild/vara-runtime/vara_runtime.compact.compressed.wasm PRINT_SCALE=1 cargo run -p gsdk-api-gen --release > "artifact/production_vara_runtime_v$VARA_SPEC_metadata.scale"
```
Comment (clearloop) [inline .github/workflows/release.yml:71]:
```suggestion
          RUNTIME_WASM=target/production/wbuild/vara-runtime/vara_runtime.compact.compressed.wasm PRINT_SCALE=1 cargo run -p gsdk-api-gen --release > "artifact/production_vara_runtime_v${VARA_SPEC}_metadata.scale"
```

Comment (clearloop) [reply .github/workflows/release.yml:71]: shall we wrap brackets for these to make it cleaner ?

**Review by clearloop — APPROVED** [review]

**Review by breathx — APPROVED** [review]

**Pull request merged**